src/styles/style.css

```diff
@@ -0,0 +1,21 @@
+
+@import "./starsPlusReviews.css";
+@import "./price.css";
+@import "./button.css";
+@import "./card.css";
```

Comment:
you should have only 2 files, one for `card` block and its elements, and the other for `stars` block